src/coupon/index.ts

```diff
@@ -2,6 +2,7 @@ export * from './coupon-actions';
 export * from './gift-certificate-actions';
 
 export { default as Coupon } from './coupon';
+export { default as CouponErrorResponseBody } from './errors/coupon-error-response-body';
```

Comment:
Can we have a `coupon/errors/index.ts` file? And export `CouponErrorResponseBody` from there?